### DISCUSSION TITLE
Unable to inference on Quantized 70B Model using llama.cpp

### DISCUSSION THREAD

**Discussion by vatsarishabh22** [ggml-org/llama.cpp#2575] [Q&A] [answered]
Got error : 
error loading model: llama.cpp: tensor 'layers.0.attention.wk.weight' has wrong shape; expected 8192 * 8192, got 8192 * 1024

I am using ubuntu linux

**Comment by klosax** [comment, marked as answer]
Use parameter `-gqa 8` for 70b models to work.
